### PR TITLE
improve annotation tag and fix evidence level tag

### PR DESCRIPTION
**Pull request by LeonardoGonzales**

### CODE REVIEW

File: common-rest/src/test/java/org/uniprot/api/common/repository/search/facet/FacetResponseConverterTest.java [hunk 1]
```diff
@@ -79,6 +79,36 @@ void convertFacetWithoutLabel() {
...
+        assertNull(itemValue.getLabel());
+        assertEquals("5", itemValue.getValue());
+        assertEquals(Long.valueOf(51L), itemValue.getCount());
+
+        itemValue = annotation.getValues().get(1);
+        assertNotNull(itemValue);
+        assertNull(itemValue.getLabel());
+        assertEquals("4", itemValue.getValue());
+        assertEquals(Long.valueOf(41L), itemValue.getCount());
+    }
```
Comment (eddturner) [inline common-rest/src/test/java/org/uniprot/api/common/repository/search/facet/FacetResponseConverterTest.java:110]: is it possible for there to be two values with the same count? Is it easy to add another section like lines 105-109, with itemValue.getValue() == 4 again, just to nail down that edge case?

**Review by eddturner — APPROVED** [review]
Good stuff!

File: common-rest/src/test/java/org/uniprot/api/common/repository/search/facet/FacetResponseConverterTest.java (hunk 1 above)
Comment (LeonardoGonzales) [reply common-rest/src/test/java/org/uniprot/api/common/repository/search/facet/FacetResponseConverterTest.java:110]: It is not possible because we populate it in a fake mocked config.
